django/views/templates/default_urlconf.html

```diff
@@ -1,19 +1,16 @@
 {% load i18n %}
 <!doctype html>
 {% get_current_language_bidi as LANGUAGE_BIDI %}
```

Comment:
You'll need to add `{% get_current_language as LANGUAGE_CODE %}` as we cannot guarantee that `django.template.context_processors.i18n` will be enabled.

Reply:
Oh, I totally missed this. Will add later today!